pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -1418,6 +1053,13 @@ class Timestamp(_Timestamp):
         # define it here instead
         return self + other
 
+    def _create_timedelta(self, *args, **kwargs):
+        """
+        Helper to create a Timedelta so that the
+        Timedelta class doesn't have to be imported elsewhere
+        """
+        return Timedelta(*args, **kwargs)
```

Comment:
I'd rather just have the runtime import in `TImedelta.resolution` (I think the only place this is used).  Or for that matter just move `resolution` from `_Timestamp` to `Timestamp`.

Reply:
It's used in two places in `_Timestamp`, `_Timestamp.__sub__` and `_Timestamp.resolution`.  If we want to go down the runtime import path, we could either add the `Timedelta` import in both functions in `_Timestamp`, or add a `_Timestamp` import into `timedeltas._binary_op_method_timedeltalike`

As for moving the `_Timestamp` functions to `Timestamp`, `_Timestamp.resolution` can be moved with no issues.  However, `_Timestamp.__sub__` can not.  Not sure where the problem is there, but if you want I can try to figure that one out.

Reply:
right that's what I mean, it doens't need to be in ``pandas/_libs/tslibs/__init__.py``, rather just import it directly where needed. its ok where its living.